frontend/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -198,6 +214,10 @@ export function* sendRequestToServer(tracingType: "skeleton" | "volume"): Saga<v
         window.onbeforeunload = null;
         yield* call(alert, messages["save.failed_simultaneous_tracing"]);
         location.reload();
+        yield* call(
+          [ErrorHandling, ErrorHandling.notify],
+          new Error("Saving failed due to '409' status code"),
+        );
```

Comment:
This should probably be called before the `location.reload()`, shouldn't it. I'd say even before the alert so the request won't get canceled by the reload (does that happen? :thinking:)

Reply:
Did you check whether these requests go through? I fear that the reload may abort them, but maybe that doesn't happen. To be on the safe side, you could move it even before the alert call a line above.

Reply:
You're right, the request is cancelled. I followed your suggestion and moved it which seems to work :)